### DISCUSSION TITLE
TinyMCE changing home-forward-slash to absolute URL of the website

### DISCUSSION THREAD

**Discussion by olivdee** [joomla/joomla-cms#38882] [General]
I was wondering if this is the intended behaviour, and if yes, why?
On a J!4.2.2 clean install (live online server), I create a custom module, and inside I add a text with a link to home like so "/". After saving, this link has been changed to the absolute URL of the website. Strangely, this does not happen on a clean install locally on XAMPP... Also, if I disable all editors or use CodeMirror, the "/" is being kept, which leads me to the conclusion, that this replacement is happening in TinyMCE. See also the attached animated GIF. 

![J!4 2 2_TinyMCE-changing-home-foward-slash-to-absolute-URL](https://user-images.githubusercontent.com/2912892/193342914-5a2b8a16-2210-4b75-a6c9-1af67871c039.gif)

**Comment by brianteeman** [comment]
what is the setting in the plugin

![image](https://user-images.githubusercontent.com/1296369/193347283-357ed050-5f8c-41d4-b39a-b84b1f5a2df6.png)

**Reply by olivdee** [reply]
On all 3 clean test installations (2x online server, 1x local XAMPP) = Relative
Working only on local/XAMPP without replacing "/" with absolute website path.

**Reply by olivdee** [reply]
If that helps or if relevant:
the only difference I can see so far is
- on both 2 online servers: PHP Version 8.0.23
- on local XAMPP: PHP Version 8.0.17
